refocus-bdk-server.js

```diff
@@ -39,7 +39,7 @@ const MIN_POLLING_REFRESH = 5000;
 const MIN_HEARTBEAT_TIMER = 60000;
 /* eslint-disable no-process-env */
 /* eslint-disable no-implicit-coercion*/
-const MAX_RETRIES = process.env.MAX_RETRIES || 3; // eslint-disable-line
+const ZERO = 0; // eslint-disable-line
```

Comment:
Would you mind to change to DEFAULT_RETRIES or something like that? If we need to change that in the future, let say, to ONE... ZERO=1 wouldn't make sense.

Its better when we add more meaning to that - otherwise we can disable lint and add 0 to all the calls

Reply:
i dont think we should ever pass anything other than ZERO in here as a number of retries but i do agree on better naming could we split difference and call it ZERO_RETRIES?